### ISSUE TITLE
Various errors

### ISSUE THREAD

**Issue by djozz911** [MatthieuBonne/CQAutomater#68]
Here are some entries I have in Actionlog :

15/06/2020 10:55:48
	URI non valide : La chaîne de l'URI est trop longue.
It appears every 2 minutes or so.

I had this entry last week on 06/06. Looks like it's on Flash Tourny day.

In the error log : 

15/06/2020 05:59:30
	Error SJ we shouldn't be running SJ, wtf ? [0,1590347302,3,3,3,1447]
15/06/2020 06:01:18
	Error flash error before sendFlashRegister {"id":3558,"hero":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,38,0,0,0,0,0,0,22,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,74,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,42,19,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,51,0,0,0,0,0,0,0,7,0,0,0,0,0,0,0,0,0,0,0,20,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,10,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,55,0,0],"followers":435110,"joined":false,"players":23,"promo":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]}

**Comment by djozz911** [comment]
Here the last error log entry with latest commit : 

15/06/2020 13:20:14
	Error updateFlashHistory  URI non valide : La chaîne de l'URI est trop longue. ---    à System.UriHelper.EscapeString(String input, Int32 start, Int32 end, Char[] dest, Int32& destPos, Boolean isUriString, Char force1, Char force2, Char rsvd)
   à System.Uri.EscapeDataString(String stringToEscape)
   à System.Net.Http.FormUrlEncodedContent.Encode(String data)
   à System.Net.Http.FormUrlEncodedContent.GetContentByteArray(IEnumerable`1 nameValueCollection)
   à System.Net.Http.FormUrlEncodedContent..ctor(IEnumerable`1 nameValueCollection)
   à CQFollowerAutoclaimer.PFStuff.updateFlashHistory(JToken json)
15/06/2020 13:20:20
	Error updateFlashHistory  URI non valide : La chaîne de l'URI est trop longue. ---    à System.UriHelper.EscapeString(String input, Int32 start, Int32 end, Char[] dest, Int32& destPos, Boolean isUriString, Char force1, Char force2, Char rsvd)
   à System.Uri.EscapeDataString(String stringToEscape)
   à System.Net.Http.FormUrlEncodedContent.Encode(String data)
   à System.Net.Http.FormUrlEncodedContent.GetContentByteArray(IEnumerable`1 nameValueCollection)
   à System.Net.Http.FormUrlEncodedContent..ctor(IEnumerable`1 nameValueCollection)
   à CQFollowerAutoclaimer.PFStuff.updateFlashHistory(JToken json)
15/06/2020 13:20:38
	Error Sending Flash lineup [-33,-72,24,21,-173,37,21,-149,-161,43,-97,1,49,-3,-69,-60,11,-145] --- {"id":3567,"hero":[0,88,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,86,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,27,0,0,0,0,0,0,0,0,3,0,0,29,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,23,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,5,0,0,0,4,0,0,0,0,0,0,0,0,0,0,0,6,0,0,0,0,0,0,0,0,0,0,0,19,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"followers":5452787,"joined":false,"players":33,"promo":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]}

edit : in the mean time, I got succesfully registered into Flash Tournament
edit 2 : won't register after the end of previous tournament
15/06/2020 13:51:10
	Error updateFlashHistory  URI non valide : La chaîne de l'URI est trop longue. ---    à System.UriHelper.EscapeString(String input, Int32 start, Int32 end, Char[] dest, Int32& destPos, Boolean isUriString, Char force1, Char force2, Char rsvd)
   à System.Uri.EscapeDataString(String stringToEscape)
   à System.Net.Http.FormUrlEncodedContent.Encode(String data)
   à System.Net.Http.FormUrlEncodedContent.GetContentByteArray(IEnumerable`1 nameValueCollection)
   à System.Net.Http.FormUrlEncodedContent..ctor(IEnumerable`1 nameValueCollection)
   à CQFollowerAutoclaimer.PFStuff.updateFlashHistory(JToken json)

**Comment by Vitae666** [comment]
> Here are some entries I have in Actionlog :
> 
> 15/06/2020 10:55:48
> URI non valide : La chaîne de l'URI est trop longue.
> It appears every 2 minutes or so.
> 
> I had this entry last week on 06/06. Looks like it's on Flash Tourny day.
> 
> In the error log :
> 
> 15/06/2020 05:59:30
> Error SJ we shouldn't be running SJ, wtf ? [0,1590347302,3,3,3,1447]
> 15/06/2020 06:01:18

Yeap, I reported the URI error in my "Might as well share my logs" post.
I agree, it's got to be the Flash reporting lineups thing.

The SJ error is not in the lastest commit. He commented that line out. You are on an older version.

**Comment by Vitae666** [comment]
From 6/15/2020 1:54:56AM to 5:55:23 AM every 2-3 mins I'd also get the `Error flash error before sendFlashRegister` error
For some reason it took a break for an hour and then at 6:44:32 AM, 7:17:10 AM, and 8:02:01 AM I got `Error Sending Flash lineup`
But it's not 2:29PM and it hasn't tried to do anything again.
Thing is that according to History, I did enter those 3 tourneys. 
[ErrorLog.txt](https://github.com/MatthieuBonne/CQAutomater/files/4782209/ErrorLog.txt)
![image](https://user-images.githubusercontent.com/9310914/84693202-25b4c780-af15-11ea-9150-8516385183fc.png)

_**EDIT:**_
BTW, just realized that you'd made updates BEFORE my comment. Sorry about that. Hadn't realized. Was late for work and just typed, not looked.
